doc/classes/Node.xml

```diff
@@ -645,199 +676,196 @@
 			<param index="0" name="node" type="Node" />
 			<param index="1" name="keep_groups" type="bool" default="false" />
 			<description>
-				Replaces a node in a scene by the given one. Subscriptions that pass through this node will be lost.
-				If [param keep_groups] is [code]true[/code], the [param node] is added to the same groups that the replaced node is in.
-				[b]Note:[/b] The given node will become the new parent of any child nodes that the replaced node had.
-				[b]Note:[/b] The replaced node is not automatically freed, so you either need to keep it in a variable for later use or free it using [method Object.free].
+				Replaces this node by the given [param node]. All children of this node are moved to [param node]. All signal connections to this node's [Callable]s are also moved to [param node]'s callables.
```

Comment:
> All signal connections to this node's [Callable]s are also moved to [param node]'s callables.

This is not true:

```gdscript
# Signal `node.signal` was connected to `before.callable`.
node.signal.connect(before.callable)
assert(node.signal.is_connected(before.callable))

# Replace!
before.replace_with(after)

# Signal `node.signal` is still connected to `before.callable`, not `after.callable`.
assert(node.signal.is_connected(before.callable))
assert(not node.signal.is_connected(after.callable))
```

Reply:
Odd. Last time I checked, it used to work that way. Stripped this line.